.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+# Description
+<!--- 
+- a short summary of the pull request (PR)
+- acceptance criteria that you did not cover and why you did not cover these
+- open questions
+-->
+
+## Ticket Link
```

Comment:
Should we change this to just ticket number?, otherwise people just copy paste the full jira link here. We wanted to avoid putting internal RKI links on github.

Reply:
done in e64d48a5cf2a35cd58b8ed44aca4c004f896a06a